src/main/resources/application-custom.yaml

```diff
@@ -25,6 +25,9 @@ spring:
     # database connection pool size
     hikari:
       maximum-pool-size: 25
+      connection-test-query: SELECT 1
+      connection-timeout: 60000
```

Comment:
After reading some of the docs, I don't think this is the right solution. This could lead to some very slow calls to the fhir server.

From the docs:

> This property controls the maximum number of milliseconds that a client (that's you) will wait for a connection from the pool. If this time is exceeded without a connection becoming available, a SQLException will be thrown. Lowest acceptable connection timeout is 250 ms. Default: 30000 (30 seconds)